pkg/logql/metrics.go

```diff
@@ -264,46 +259,71 @@ func RecordSeriesQueryMetrics(
 		latencyType = latencyTypeSlow
 	}
 
-	// we also log queries, useful for troubleshooting slow queries.
-	level.Info(logger).Log(
+	shard := extractShard(shards)
+
+	logValues := make([]interface{}, 0, 15)
+	logValues = append(logValues,
 		"latency", latencyType,
 		"query_type", queryType,
+		"start", start.Format(time.RFC3339Nano),
+		"end", end.Format(time.RFC3339Nano),
+		"start_delta", time.Since(start),
+		"end_delta", time.Since(end),
 		"length", end.Sub(start),
 		"duration", time.Duration(int64(stats.Summary.ExecTime*float64(time.Second))),
 		"status", status,
 		"match", PrintMatches(match),
-		"splits", stats.Summary.Splits,
-		"throughput", strings.Replace(humanize.Bytes(uint64(stats.Summary.BytesProcessedPerSecond)), " ", "", 1),
-		"total_bytes", strings.Replace(humanize.Bytes(uint64(stats.Summary.TotalBytesProcessed)), " ", "", 1),
-		"total_entries", stats.Summary.TotalEntriesReturned,
```

Comment:
splits is still useful for series queries

Reply:
I _think_ this stat was not captured. Will check again, same for the other instance

Reply:
Confirmed, in neither case is `stats.Summary.Splits` recorded

Reply:
Apparently I was wrong; I added them back 👍